14/umbraco-cms/reference/security/external-login-providers.md

```diff
@@ -177,9 +177,9 @@ using Umbraco.Cms.Web.BackOffice.Security;
 
 namespace MyUmbracoProject.CustomAuthentication;
 
-public class ProviderBackOfficeExternalLoginProviderOptions : IConfigureNamedOptions<BackOfficeExternalLoginProviderOptions>
+public class GenericBackOfficeExternalLoginProviderOptions : IConfigureNamedOptions<BackOfficeExternalLoginProviderOptions>
```

Comment:
the code title needs to be changed as well to GenericBackOfficeExternalLoginProviderOptions